server/src/test/java/org/elasticsearch/index/query/SpanContainingQueryBuilderTests.java

```diff
@@ -89,5 +91,92 @@ public void testFromJson() throws IOException {
 
         assertEquals(json, 2, ((SpanNearQueryBuilder) parsed.bigQuery()).clauses().size());
         assertEquals(json, "foo", ((SpanTermQueryBuilder) parsed.littleQuery()).value());
+        assertEquals(json, 2.0, parsed.boost(), 0.0);
+    }
+
+    public void testFromJson_withNonDefaultBoost_inBigQuery() {
```

Comment:
can we NOT use "_" here, just a camel case!

Reply:
Sure, fixed in 043020b786ec289d20a6d72b3cf50af37831fb37.